.eslintrc.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+  'plugins': ['lodash-fp', 'react'],
+  'extends': 'react-app',
+  'globals': {
+    'SATURN_VERSION': false,
```

Comment:
Will these eventually be JADE things?

Reply:
Yes, eventually once we remove the instance of SATURN_VERSION, etc from the code.